tailwind.config.js

```diff
@@ -14,6 +14,9 @@ export default {
       "accent-color4": "#174444",
       "static-color": "#F7F7F7",
     },
+    fontFamily: {
+      roboto: ["Roboto"],
```

Comment:
Use the font family from the variables.css, dont used by Tailwind; it is used for maintenance if we want to alter the typeface.